### ISSUE TITLE
on k8s plugins does not work properly

### ISSUE THREAD

**Issue by httpsOmkar** [overhangio/tutor#292]
**Bug description**
When you install the edx with k8s and enable plugins in discovery, ecommerce.

```
error: NewResMapFromConfigMapArgs: NewResMapFromConfigMapArgs: file sources: [plugins/ecommerce/apps/settings/tutor.py]: evalsymlink failure on '/home/omkar/.local/share/tutor/env/plugins/ecommerce/apps/settings/tutor.py' : lstat /home/omkar/.local/share/tutor/env/plugins/ecommerce/apps/settings: no such file or directory
```

**How to reproduce**

**Environment**
Arch Linux
tutor, version 3.11.3

**Additional context**
I think issues is in the `~/home/omkar/.local/share/tutor/env/kustomization.yml`
when I open this file the contents are like 

```yaml
---
apiVersion: kustomize.config.k8s.io/v1beta1
kind: Kustomization
resources:
- k8s/namespace.yml
- k8s/deployments.yml
- k8s/ingress.yml
- k8s/services.yml
- k8s/volumes.yml


# namespace to deploy all Resources to
namespace: openedx

# labels added to all Resources
# https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/
commonLabels:
  app.kubernetes.io/instance: openedx-TvxbyecWjczTnyIXvEQG0uIM
  app.kubernetes.io/version: 3.11.3
  app.kubernetes.io/part-of: openedx
  app.kubernetes.io/managed-by: tutor
  

configMapGenerator:
- name: openedx-settings-lms
  files:
  - apps/openedx/settings/lms/__init__.py
  - apps/openedx/settings/lms/development.py
  - apps/openedx/settings/lms/production.py
- name: openedx-settings-cms
  files:
  - apps/openedx/settings/cms/__init__.py
  - apps/openedx/settings/cms/development.py
  - apps/openedx/settings/cms/production.py
- name: openedx-config
  files:
  - apps/openedx/config/cms.auth.json
  - apps/openedx/config/cms.env.json
  - apps/openedx/config/lms.auth.json
  - apps/openedx/config/lms.env.json
- name: nginx-config
  files:
  - apps/nginx/cms.conf
  - apps/nginx/extra.conf
  - apps/nginx/lms.conf
  - apps/nginx/tutor.conf
- name: mysql-config
  env: apps/mysql/auth.env
- name: discovery-settings
  files:
    - plugins/discovery/apps/settings/tutor.py #  <---- THIS FILE DOES NOT EXITS
```
Where the this file is not exits. instead `/home/omkar/.local/share/tutor/env/plugins/discovery/apps/settings/tutor/`
`development.py  __init__.py  production.py`
This file exits

**Comment by regisb** [comment]
Hi @httpsOmkar! This indeed looks like a bug in the discovery plugin. I'll make a new release of this plugin (v0.1.11).

**Comment by regisb** [comment]
Fixed in v0.1.12 by:
https://github.com/overhangio/tutor-discovery/commit/d76040dff51f9b4434e94a996d8edfc59d53ef7e
https://github.com/overhangio/tutor-discovery/commit/b22b0000c417e63d0f30be70daa48eb41305a7ad
